install/restart_scripts/renew_ra_cert

```diff
@@ -28,14 +28,14 @@ import shutil
 import traceback
 
 from ipalib.install.kinit import kinit_keytab
-from ipalib import api
-from ipaserver.install import certs, cainstance, dogtaginstance
+from ipalib import api, x509
+from ipaserver.install import certs, cainstance
 from ipaplatform.paths import paths
 
+from cryptography.hazmat.primitives import serialization
```

Comment:
Nitpick: please place 3rd party module imports between standard module imports and IPA module imports.